app/models/comment.rb

```diff
@@ -1,4 +1,15 @@
+require 'csv'
+
 class Comment < ApplicationRecord
   include Visible
   belongs_to :post
+
+  def self.to_csv
```

Comment:
Doesn't that define a class method? How does calling `comments.to_csv` work then? Perhaps some Rails magic? :D